lib/src/internals/base_repository.dart

```diff
@@ -37,12 +37,8 @@ abstract class BaseRepository implements ModelRepository {
     return query.apply(db, params);
   }
 
-  Future<T> transaction<T>(Runnable<T> runnable) {
-    return db.runTransaction(runnable);
-  }
-
   @override
   Future<void> run<T>(Action<T> action, T request) {
-    return transaction(() => action.apply(db, request));
```

Comment:
What is the reason for removing all the transactions? Data modifications should always be atomic and the package may run multiple queries for a insert or delete action.

Reply:
For various reasons:
- It should be the user who wants this behavior.
- The user so can perform multiple queries in a single transaction
- The Session class does not have the methods to execute transactions

Thanks to this change you can now write:
```dart
  final db = Database(...);
  await db.run((session) async {
    await session.companies.deleteOne('abc');

    await session.companies.insertOne(CompanyInsertRequest(
      id: 'abc',
      name: 'Minga',
      addresses: [],
    ));

    await session.accounts.deleteMany([0, 1, 2]);
  });
```
Remember: when you open one transaction you cannot open another so it must be the user who wants to open a transaction.

Reply:
Ok thanks for the explanation